src/components/PersonaDialog.vue

```diff
@@ -103,4 +103,8 @@ watch(
       ipfs_hash: 'QmQMBfgnmuxcQ4kptR1oPE9guYxG13GpASjYVeFQSxNxjE',
     }),
 );
+watch(
+  () => name.value,
+  () => (props.basePersona.label = name.value),
```

Comment:
you should not mutate the prop, and a watch is not even needed here you can pass the label directly in the event emitted.
I'll update this later today